src/includes/performance/traces-sampler-as-sampler/dotnet.mdx

```diff
@@ -13,7 +13,7 @@ SentrySdk.Init(options =>
     {
         var ctx = context.CustomSamplingContext;
 
-        return ctx switch
+        return ctx["url"] switch
```

Comment:
wouldn't this throw though if `url` isn't there? Not sure this is good sample code

Reply:
I just adapted the sample from here: https://github.com/getsentry/sentry-docs/blob/caf5692e6dcd804b33c8832125269b3f9d1e457c/src/includes/performance/traces-sampler-as-sampler/java.mdx

Reply:
Doesn't need to look the same, has to make sense here.
People will 100% copy paste this code into their apps and it'll crash when there's no `url` in the context object so we need to be mindful of our samples

Reply:
In that case...

```suggestion
        return ctx.GetValueOrDefault("url") switch
```